test-configs.yaml

```diff
@@ -751,35 +735,57 @@ device_types:
       - whitelist: {defconfig: ['defconfig']}
       - blacklist: {kernel: ['v3.']}
 
-  qemu_arm:
-    type: qemu
+  qemu-arm-virt-gicv2:
+    type_name: qemu
```

Comment:
Sorry for the confusion with the recent changes - this should be `type: qemu`

The code has `type_name` mainly because `type()` is a built-in Python class name.